src/graphql/data/nft/Collection.ts

```diff
@@ -87,26 +86,14 @@ const collectionQuery = graphql`
   }
 `
 
-export function useLoadCollectionQuery(address?: string | string[]): void {
-  const [, loadQuery] = useQueryLoader(collectionQuery)
-  useEffect(() => {
-    if (address) {
-      loadQuery({ addresses: Array.isArray(address) ? address : [address] })
-    }
-  }, [address, loadQuery])
-}
+export function useCollectionQuery(address: string): { data: GenieCollection; loading: boolean } {
+  const { data: queryData, loading } = useCollectionQueryQuery({
```

Comment:
`useCollectionQueryQuery` is kind of odd. id probably rename to `useCollectionQuery` and then the outer hook seems more semantic to name `useCollection` since that's what you're returning here

Reply:
See comment here https://github.com/Uniswap/interface/pull/5647#discussion_r1048904357